learn/getting-started/visual-studio-code-extension/debugging.md

```diff
@@ -164,7 +166,24 @@ Visual Studio Code allows you to debug Ballerina programs through the Ballerina
 
 <!--For more information on the above features, see [Using the Debugging Features](/learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features/).-->
 
-### Using Expression Evaluation
+### Conditional Breakpoints
+
+The Ballerina debugger provides the ability of setting Ballerina expression-based conditions for regular breakpoints. 
+The debugger will suspend at the breakpoint whenever the expression evaluates to true.
+
+![Debugger Conditional Breakpoints](/learn/images/debugger-conditional-breakpoints.gif)                
+                      
+### Logpoints        
+
+Logpoints are another variant of breakpoints which does not "break" the program execution, but instead logs a message to the console. Logpoints are especially useful for injecting logging while debugging production servers that cannot be paused or stopped.
+
+Log messages can either be plain texts, or string templates which can contain expressions to be evaluated within curly braces ('{}').
```

Comment:
```suggestion
Log messages can either be plain texts or string templates, which can contain expressions to be evaluated within curly braces ('{}').
```

Reply:
@praneesha In here, the 'which can contain expressions to be evaluated within curly braces ('{}')' condition does not apply for plain texts and thats why I added the comma in there. WDYT?

Reply:
Shall we change as follows:

`Log messages can either be plain texts, or string templates, which can contain expressions to be evaluated within curly braces ('{}').`